Dockerfile

```diff
@@ -13,7 +13,7 @@ ADD ./Gemfile.lock /rails_app/
 ADD ./Jarfile /rails_app/
 ADD ./Jarfile.lock /rails_app/
 
-RUN bundle install --without development test
+RUN bundle install
```

Comment:
don't do this, it adds spring and figrake back to prod env. do we need these gems installed?